src/components.d.ts

```diff
@@ -717,6 +725,40 @@ export namespace Components {
          */
         "teaserColour": string;
     }
+    interface BiggiveIconButton {
+        /**
+          * Arrow Colour
+         */
+        "arrowColour": string;
+        /**
+          * Background Colour
+         */
+        "backgroundColour": string;
+        /**
+          * Text
+         */
+        "iconColour": string;
+        /**
+          * Text
+         */
+        "label": string;
+        /**
+          * New Tab
+         */
+        "openInNewTab": boolean;
+        /**
+          * Space below component
+         */
+        "spaceBelow": number;
+        /**
+          * Text Colour
+         */
+        "textColour": string;
+        /**
+          * URL
+         */
```

Comment:
None of the these docblocks seem to say anything that doesn't duplicate what the code says. I think we could profit by removing them. (See https://github.com/thebiggive/regression-tests/pull/84 )

```suggestion
        "arrowColour": string;

        "backgroundColour": string;

        "iconColour": string;

        "label": string;

        "openInNewTab": boolean;

        "spaceBelow": number;

        "textColour": string;

```